README.md

```diff
@@ -14,26 +15,35 @@ to provide code folding based on the tree-sitter syntax tree.
 </p>
 
 <!-- Markdown is not able to render links with unicode so after refreshing the toc, select it and:
-    `M-x regexp-replace #[^a-zA-Z] <ret> # <ret>` -->
+    `M-x regexp-replace #[^-a-zA-Z] <ret> # <ret>` -->
```

Comment:
Can you confirm the change here? It seems like you have accidentally changed the line.

Reply:
I added this extra dash because then if you run the search-replace multiple times, it won't clear out the dashes that come after the '#'. By putting it in the front of the matching group, it isn't seen as a range so it is read as a dash.

One thing I just noticed, in your emacs is the command actually `regexp-replace`? In mine, it's showing up as `replace-regexp`. I use evil mode so I I'm not super familiar with standard emacs commands and don't want to change it without checking first.

Reply:
>I added this extra dash because then if you run the search-replace multiple times, it won't clear out the dashes that come after the '#'. By putting it in the front of the matching group, it isn't seen as a range so it is read as a dash.

Thanks for the clarification! 👍 

That line was added by @mattiasdrp, but I think you are correct. It shows up `replace-regexp` and not `regexp-replace`. Can you fix that up for us? 😓 Otherwise, it should be good to merge! 😅 Thank you for your patient, sorry it took this long to get this merge!